craft_grammar/create.py

```diff
@@ -105,7 +104,7 @@ def _get_grammar_type_for(model_type: type) -> str | None:
 
         case typing.Union:
             # Type is either a Union[] or an Optional[]
-            if len(args) == 2 and type(None) in args:
+            if len(args) == 2 and type(None) in args:  # noqa: PLR2004
```

Comment:
pedantic: I prefer `# noqa: PLR2004 (magic-value-comparison)` for readability for all `noqa` comments

Reply:
Agreed - some sort of explanation is nice. I don't really care about the format, but "PLR2004" is hard to figure out what we're #noqa-ing

Reply:
good idea, updated